mirrord/agent/src/cli.rs

```diff
@@ -62,8 +71,18 @@ impl Mode {
     pub fn is_targetless(&self) -> bool {
         matches!(self, Mode::Targetless)
     }
+
+    pub(super) fn mesh(&self) -> Option<MeshVendor> {
+        match self {
+            Mode::Targeted { mesh, .. } => mesh
+                .as_ref()
+                .and_then(|mesh| MeshVendor::from_str(mesh).ok()),
```

Comment:
I don't like this - it means if someone enters wrong value it's just ignoring it..

Reply:
Oh, it panics on the wrong value, see the `from_str` implementation.